src/main/java/seedu/academyassist/model/person/SubjectEnum.java

```diff
@@ -5,14 +5,24 @@
  * Each subject has a name that can be accessed via the {@link #getSubjectName()} method.
  */
 public enum SubjectEnum {
-    /** Subject for English. */
     ENGLISH("English"),
-    /** Subject for Math. */
-    MATH("Math"),
-    /** Subject for Chinese. */
     CHINESE("Chinese"),
-    /** Subject for Science. */
-    SCIENCE("Science");
+    MALAY("Malay"),
+    TAMIL("Tamil"),
+    MATH("Math"),
+    FURTHER_MATH("Further Math"),
+    SCIENCE("Science"),
+    HISTORY("History"),
```

Comment:
Good that there are more subjects in case we become a bigger tuition centre in the future 👍